NodeManagerLibrary.h

```diff
@@ -28,8 +28,14 @@
 #define ON 1
 
 // define interrupt pins
+
+#if defined(ARDUINO_ARCH_STM32F0) || defined(ARDUINO_ARCH_STM32F1) || defined(ARDUINO_ARCH_STM32F3) || defined(ARDUINO_ARCH_STM32F4) || defined(ARDUINO_ARCH_STM32L4)
```

Comment:
May I ask you to move the entire "Chip type" section before "Constants" so you can reuse  the CHIP_STM32 define?

Reply:
Yes, it will be better.